lib/components/Text/Text.migration.md

```diff
@@ -0,0 +1,40 @@
+# Text Migration Guide
+
+## API Changes
+
+- No longer handles heading text. Use [`Heading`](https://seek-oss.github.io/braid-design-system/components/Heading) instead.
+- Deprecated `conversational`/`loud`/`intimate`/`whispering` in favour of `size={'large' | 'standard' | 'small'}` and `weight={'strong' | 'regular'}`. Please note that this is a breaking change to our design language. When migrating, try to match existing styling as much as possible in consultation with your local designer.
+- No longer renders white space below it, also removing the need for a `raw` prop. Use a [`Paragraph`](https://seek-oss.github.io/braid-design-system/components/Paragraph) component instead.
```

Comment:
Maybe soften this to something like "Consider using a `Paragraph` where required", or similar as that will not be a blanket solution.